tests/operator_test.go

```diff
@@ -1394,16 +1399,28 @@ spec:
 
 			migratableVMIs := generateMigratableVMIs(2)
 			launcherSha := getVirtLauncherSha()
-			Expect(launcherSha).ToNot(Equal(""))
+			if !flags.SkipShasumCheck {
+				Expect(launcherSha).ToNot(Equal(""))
+			}
 
 			previousImageTag := flags.PreviousReleaseTag
 			previousImageRegistry := flags.PreviousReleaseRegistry
 			if previousImageTag == "" {
 				previousImageTag, err = tests.DetectLatestUpstreamOfficialTag()
 				Expect(err).ToNot(HaveOccurred())
-				By(fmt.Sprintf("By Using detected tag %s", previousImageTag))
+				By(fmt.Sprintf("By Using detected tag %s for previous kubevirt", previousImageTag))
+			} else {
+				By(fmt.Sprintf("By Using user defined tag %s for previous kubevirt", previousImageTag))
+			}
+
+			previousUtilityTag := flags.PreviousUtilityTag
+			previousUtilityRegistry := flags.PreviousUtilityRegistry
+			if previousUtilityTag == "" && previousUtilityRegistry == "quay.io/kubevirt" {
+				previousUtilityTag, err = tests.DetectLatestUpstreamOfficialTag()
```

Comment:
I think the logic shold be to fallback to `previousImage{Registry,Tag}` if the flags for utilities are not provided. Maybe smth like bellow would be better?
```go
previousUtilityTag := flags.PreviousUtilityTag
previousUtilityRegistry := flags.PreviousUtilityRegistry
if previousUtilityTag == "" {
    previousUtilityTag = previousImageTag
}
if previousUtilityRegistry == "" {
    previousUtilityRegistry = previousImageRegistry
}
```

Reply:
@vasiliy-ul  I thought about this. The fallback you mentioned should make test continue as much as possible when previousUtilityTag is not given, right? However, if user gives PreviousReleaseTag(resides in one registry) and does not give previousUtilityTag(previousUtilityRegistry is another registry with likely different tag rules), it will end up as problematic. HDYT?

Reply:
Hm... maybe that makes sense. Though I am still not convinced about having the hardcoded `quay.io/kubevirt` registry here. Is it really needed in this check? What would be the behaviour if the previous utility tag was empty but the previous utility registry was set to smth different from quay.io? I guess with the current logic it will pass an empty tag to `generatePreviousVersionVmYamls`. Is that supposed to work?

Reply:
@vasiliy-ul  You made a point here. Remove the previousUtilityRegistry check with quay.io, will give the situation that you described a second chance to work potentially. Will change code to that.

Reply:
Change is done accordingly.

Reply:
thanks, looks good :)